### DISCUSSION TITLE
Planning applications & decisions

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#25] [Planning consideration]
# Planning applications and decisions

These are the applications for development that are submitted to planning authorities. Planning authorities assess the applications, ask the public for comment and make a decision on whether permission will be granted or not.

## What

We are looking to improve the availability of planning application and decision data at the national level. Starting with the essential elements of the application and the key decision a Local Planning Authority makes.

Planning applications are currently available - anyone can visit a local planning authority website and find planning applications - but they are not always available as data. There are services such as [Search the register of planning decisions](https://www.gov.uk/search-register-planning-decisions) but all these can do is route users to a Local Planning Authority website for them to view planning applications for an area.

Some Local Planning Authorities publish planning application data but the shape, size and frequency of that data can vary.

## Why (we are looking at it)

Consistent planning application data at a national level has the potential to unlock a lot of value in the planning system. It will make it easier for policymakers to make more informed interventions, simpler for applicants to make applications with a good chance of being granted and more likely 3rd parties will be able to create services that improve engagement with the planning system.

We also believe planning application and decision data will be linked to by a lot of the other datasets we are looking to create and open up. Therefore, it is important we at least create a minimal data standard for planning applications and decisions that other datasets can link to.

## What we're looking for

We want to know more about:

* who is currently publishing planning application data and where are they publishing it
* who would like to use the data and what do they want to use it for
* if anyone asking for this data
* what should be included in a data standard
* if any attempts to standardise the data have been tried before

**Comment by colmjude** [comment]
We have created a [draft standard for planning applications and decisions](https://digital-land.github.io/specification/specification/planning-application-decision/).

Our next step is to test this with real data published by Local Planning Authorities.

We are also researching [planning conditions](https://github.com/digital-land/data-standards-backlog/issues/4) - a dataset that links to planning applications.

**Comment by colmjude** [comment]
A few local planning authorities who publish their planning application data

* [Nottingham on open data nottingham](https://www.opendatanottingham.org.uk/dataset.aspx?id=28)
* [Camden](https://opendata.camden.gov.uk/Environment/Planning-Applications/2eiu-s2cw/explore/query/SELECT%0A%20%20%60pk%60%2C%0A%20%20%60application_number%60%2C%0A%20%20%60development_address%60%2C%0A%20%20%60development_description%60%2C%0A%20%20%60decision_type%60%2C%0A%20%20%60valid_from_date%60%2C%0A%20%20%60registered_date%60%2C%0A%20%20%60registered_in_last_7_working_days%60%2C%0A%20%20%60registered_in_last_28_working_days%60%2C%0A%20%20%60earliest_decision_date%60%2C%0A%20%20%60decision_date%60%2C%0A%20%20%60decison_level%60%2C%0A%20%20%60system_status%60%2C%0A%20%20%60system_status_change_date%60%2C%0A%20%20%60applicant_name%60%2C%0A%20%20%60ward%60%2C%0A%20%20%60conservation_areas%60%2C%0A%20%20%60neighbourhood_areas%60%2C%0A%20%20%60case_officer%60%2C%0A%20%20%60case_officer_team%60%2C%0A%20%20%60responsibility_type%60%2C%0A%20%20%60comment%60%2C%0A%20%20%60full_application%60%2C%0A%20%20%60application_type%60%2C%0A%20%20%60easting%60%2C%0A%20%20%60northing%60%2C%0A%20%20%60longitude%60%2C%0A%20%20%60latitude%60%2C%0A%20%20%60spatial_accuracy%60%2C%0A%20%20%60last_uploaded%60%2C%0A%20%20%60location%60%2C%0A%20%20%60socrata_id%60%2C%0A%20%20%60organisation_uri%60%2C%0A%20%20%60%3A%40computed_region_hxwp_gyfc%60%2C%0A%20%20%60%3A%40computed_region_6i9a_26nj%60%0AORDER%20BY%20%60case_officer%60%20DESC%20NULL%20FIRST/page/filter)
* [Adur](https://www.adur-worthing.gov.uk/datasets/planning/adur-planning.csv) and [Worthing](https://www.adur-worthing.gov.uk/datasets/planning/worthing-planning.csv) have published planning application data ([documentation page](https://www.adur-worthing.gov.uk/datasets/))

**Comment by colmjude** [comment]
The LGA has previous drafted [a standard for planning applications](https://github.com/esd-org-uk/schemas/tree/master/PlanningApplications).

**Comment by RichLimbrick** [comment]
This is looking really good. One thing that is important is creating a field for linking to associated applications. This is particularly relevant for applications that:

1. Discharge a condition associated with an existing permission (approval of details or AOD),
2. Make [Minor Material Amendments (MMAs)](https://www.gov.uk/guidance/flexible-options-for-planning-permissions) to an existing permission (a s73 application), or
3. Make [Non-Material Amendments (NMAs)](https://www.gov.uk/guidance/flexible-options-for-planning-permissions) to an existing permission (a s96A application).

It makes sense to me that this would be an additional field within the planning-application table.
This data standard could help to map and track permissions and to understand quickly which scheme is actually being implemented (very important for delivery and monitoring purposes). This would also link in with conditions work you are doing but that is probably not for now.

**Comment by colmjude** [comment]
Thanks for that @RichLimbrick, that is really useful to know. And definitely feels like that should be part of the standard.

Just to clarify can an application only be amended once? As in application Y amends application X.
Then if there is a further amendment, it is to this newer application? I.e. application Z now amends application Y?
So it is like a chain? X -> Y -> Z

Or can multiple applications reference the same original application?
In particular, I’m thinking about case 1 - discharging conditions. If multiple conditions exist on an application, can they be discharged one by one? 

Whilst on discharging conditions, we’ve heard conditions can elapse after a set amount of time. Is that true?
And in those cases is the application amended to say the condition no longer applies or is it just a case of checking the condition and seeing it has elapsed?

I hope those questions make sense.

**Comment by colmjude** [comment]
We recently created a [proof of concept prototype](https://planning-application-prototype-876601a5bfae.herokuapp.com/).

We were at the stage where we wanted to look at some real data, to see how the current draft standard stacks up, and to see how far away what is currently published is from the standard we are proposing. The prototype will also allow us to have better, more focused conversations.

## What we did

We collected data from 3 LPAs:

- Nottingham
- Adur and Worthing
- Camden

We then built a UI to explore the data. We based the UI on what a basic service built on top of an open register of planning applications could look like.

## What we learned

It was a good start. We collected over 100k planning application records.
The records we collected might not have all the data we'd like but there is still a record for each planning application containing enough information to do useful things.

It was difficult to aggregate the data because of a lack of consistency.
Some consistency exists because of the work the LGA did defining [a schema for planning applications](https://schemas.opendata.esd.org.uk/PlanningApplications/LocalOpenDataIncentiveSchemePlanningApplicationsSchemaGuidance.pdf). That is really useful, but everyone doesn't follow it.
Some LPAs also publish multiple sources - both CSVs and GeoJSON. On the one hand, this is helpful because it meant we could extract geometries (polygons or points). However, the properties in the GeoJSON and the columns in the CSV don't always match. When they don't match we have to create mappings between the 2.
Mapping fields isn't difficult to do technically, but it requires us to understand the intent behind the property. Sometimes it is obvious, and sometimes it's not. Access to domain expertise would help, but that won't always be possible, which is why it is so important to push for consistency.

Here is the list of properties we found in the published data.
![The set of properties we found in the data](https://github.com/digital-land/data-standards-backlog/assets/299875/6efacbe4-ee89-42cb-a16b-b4e8ccaa776d)

The published data we found is often one wide record - one record contains all the properties. In contrast, we'd like to split the record into parts. Adopting a more relational approach to the shape of the data gives it more flexibility over time. For example, publishers can add a new row to the log if a new activity or event becomes a desirable thing to record. This is easier than constantly adding and removing properties from the top-level planning application record.

Ultimately, we had to write some bespoke code to extract the dates and make the beginnings of a log. It was helpful for this exercise but is not a sustainable approach as this scales.

It is worth noting that we collected and created a substantial number of records. We have over 100k planning application records and roughly 400k log entries. If we get similar numbers from all LPAs then we are talking about a significant amount of data. We might need to work out alternative ways to collect this data. Collecting the data as a feed might make more sense than collecting all of an LPA's planning application records in one go. It is something we need to explore.

When building the filtering options, we used the log to filter the applications submitted after a given date. This worked fine but it does mean a data consumer needs both datasets to be able to query the data by dates. Maybe that is ok, maybe not. It is something for us to think about.

We've hosted the [prototype](https://planning-application-prototype-876601a5bfae.herokuapp.com/) on Heroku so feel free to look and let us know what you think.

**Comment by RichLimbrick** [comment]
> Thanks for that @RichLimbrick, that is really useful to know. And definitely feels like that should be part of the standard.
> 
> Just to clarify can an application only be amended once? As in application Y amends application X. Then if there is a further amendment, it is to this newer application? I.e. application Z now amends application Y? So it is like a chain? X -> Y -> Z
> 
> Or can multiple applications reference the same original application? In particular, I’m thinking about case 1 - discharging conditions. If multiple conditions exist on an application, can they be discharged one by one?
> 
> Whilst on discharging conditions, we’ve heard conditions can elapse after a set amount of time. Is that true? And in those cases is the application amended to say the condition no longer applies or is it just a case of checking the condition and seeing it has elapsed?
> 
> I hope those questions make sense.

Hi @colmjude - I have started putting this together. It's a work in progress but hope it helps with the above questions. Happy to keep discussion on this especially if I should map it out in a different way for you or with better explanations.
[Miro board - Planning applications - relationships between applications and decisions](https://miro.com/app/board/uXjVMxfDy8Y=/?share_link_id=200078533113)

**Comment by rcr-os** [comment]
My recommendation would be to add additional fields that help to identify the type of development being planned (residential or commercial) and then specifically identify:
- The number of new dwellings (in a residential development)
- The use/class of development for commercial - e.g. industrial, retail, leisure etc.

This data will be useful to major infrastructure companies in the water, energy and telecoms sector. Analysis of this type of data can help these infrastructure companies plan their future infrastructure investments and operations better. Happy to elaborate further.

**Comment by SarahPurssellDfT** [comment]
Hi, I'm the Product Manager for a service for DfT which will allow people to apply for stopping up orders. In order to apply they have to have submitted a planning application so it would be great to ask them for a planning application reference number and pull back: 

- date of application/approval/appeals
- planning status
- planning authority 
- planning type
- site plans
- planning decision/application

This would help our external users because they would only give the government this information once and it would ensure consistency between DfT information and the information held by the planning authority and help DfT process stopping up order applications quicker.

**Reply by colmjude** [reply]
Hi [SarahPurssellDfT](https://github.com/SarahPurssellDfT), thats really helpful, thank you.

Can I ask what a stopping up order is? Is there anywhere you can point me where I can read more about them?

**Comment by colmjude** [comment]
# Existing sources of planning application data

As well as the [planning authorities who publish their application data](https://github.com/digital-land/data-standards-backlog/discussions/25#discussioncomment-6678175) we have found the following sources of planning application data:

* [LGA aggregator](https://aggregator.opendata.esd.org.uk/?schemaURI=http://schemas.opendata.esd.org.uk/PlanningApplications) (has 10,000s of planning application records, not sure how up to date it is)
* [uk_planning_scraper](https://github.com/adrianshort/uk_planning_scraper) - a ruby gem for scraping planning app data from LPA sites
* [UK PlanIt](https://www.planit.org.uk/) - details of 19000000+ applications, has an API
* [Build with tract planning application scaper](https://github.com/buildwithtract/planning-applications) 

Other avenues to explore include:

* [National statistics published by the government](https://www.gov.uk/government/statistics/planning-applications-in-england-july-to-september-2024) - we can explore getting the data behind the statistics
* [another statistical-based publication from MHCLG](https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Fassets.publishing.service.gov.uk%2Fmedia%2F678517f03a9388161c5d2322%2FTables_P120_to_P138_MJPSI_PIPTDCC_Final.ods&wdOrigin=BROWSELINK)
* Individual LPAs publish their application information on maps (for example, [Barnet](https://publicaccess.barnet.gov.uk/online-applications/spatialDisplay.do?action=display&searchType=Application)) - can we get to the data behind it?

**Comment by colmjude** [comment]
# Revisiting the planning application and decisions specification

We’ve begun revisiting our [planning application and decision data specifications](https://digital-land.github.io/specification/specification/planning-application-decision/). This was originally drafted in 2022 and needs reworking to incorporate everything we’ve learned from the [planning application submissions work](https://design.planning.data.gov.uk/advisory-group). Here’s a quick overview of where we are, how we’re approaching it, and where you come in.

### Where we are now

Working on aligning both the planning application submission and decision specifications. Most of the data at the point of decisions is the same or an updated version of what was originally submitted so we've making sure these two are closely aligned with the distinction:

•	Submission specs define the data needed to validate, assess and decide on an application.
•	Decision specs capture what’s needed after a decision is made; so you can record, monitor and report on it.

### Our approach (start small and build with purpose)

Here are some principles we are following for the decisions specification(s)

1.	Include only what we’re confident adds value
     No filler data. Every field must have a clear use case (need)
2.	Validate every addition with real needs
    If you need dwellings-count or developer engagement details, tell us, and we can include that
3.	Break things into connected, reusable parts
    Smaller, linked datasets are easier to maintain and mandate.
4.	Avoid inferrable fields
    Let systems compute what they can, keeping the data clean, consistent and low-burden

### What you’ll see

This is a very early draft of our core parts and fields. We’ll update and expand it based on your feedback and user research.

![CleanShot 2025-06-20 at 16 00 03@2x](https://github.com/user-attachments/assets/0c73539e-5d31-41b1-895f-bec69731aa4f)

### What’s next

1.	User research to dig into post-decision needs
	•	Who uses this data? (e.g. GLA, MHCLG, developers, consultees)
	•	How do they use it? (dashboards, monitoring reports, community engagement, policy analysis…)
2. Evolve our [project page](https://design.planning.data.gov.uk/advisory-group) into a single project page that covers both submission and decision specifications.
3.	Flesh out the specifications. We’ll share updates early and often, so the specifications grow in line with your needs

### Your input

For these specifications to work we need your input.

Please get in touch if you:
	•	would like to join our user research sessions
	•	have examples or suggests of use cases for the planning application & decision data

**Reply by colmjude** [reply]
## A couple of specific questions

1. **Officer’s report**. Should we model this as its own dataset (with metadata fields) rather than just lumping it in the “documents” bucket?
2. **Planning conditions**. We’re testing a [draft specification](https://digital-land.github.io/specification/specification/planning-condition/) with a few LPAs (thank you 🙏). One potential addition keeps cropping up: should we add a field to record which Permitted Development Rights are affected or removed by a condition? If you’ve seen conditions that reference PDRs, please share your examples! (there is [a discussion thread specifically for planning conditions](https://github.com/digital-land/data-standards-backlog/discussions/40)).

**Reply by BobbuTPX** [reply]
Hi Colm,

The DPR team had a look through the core fields early draft, and we've pulled out some observations:

### Comments
We notice that comments aren't included in the specification, and we think they would be an important inclusion. We know that publishing comments individually is not a legal requirement, but that addressing all points raised in comments is a legal requirement of the officers report. We know it might be a bit much for the minimal goal of this spec, but believe the inclusion of comments would be extremely useful part of the specification, given how crucial they are to consultations. They are also often stored as documents at present, and moving them towards being data would make a very significant improvement to the problem of document storage.

### Statuses
The [status codes](https://dataset-editor.planning.data.gov.uk/dataset/planning-application-status/finder) seem to be incomplete.  We are unsure if they are only meant for public consumption or back office use? If it's for public use, then `request-missing-document` and `local-planning-authority-document-check` are not things the public has an understanding or awareness of, based on our research - so many be more confusing than helpful. Statuses relating to committee sessions are also missing (we haven't quite drilled down into this yet ourselves, either, so we don't have specific recommendations yet). However, if these statuses are for back office usage, then things like validation of an application and officer recommendation steps are missing. 

### Decision notice
We are still working on decision notice legal requirements ourselves, but we believe that you are missing the reason the application is granted or refused. This is present on many of the decision notices we've looked at.

### Application log
We were not sure what the difference is between the first two fields on the log are; `application` and `application-reference`? This would suggest that they are either redundant, or need a clearer explanation for what they are. `planning-application-event` appears to have the wrong description on the mural, too.

### Amendments applications
It was raised that there are applications which can be submitted that are amendments to an existing application, which mean connecting applications together at this phase seems quite important. [This mural](https://miro.com/app/board/uXjVMxfDy8Y=/) goes into detail about how these kinds of amendment applications can impact prior applications, and make it complicated to track compliance with permissions given. I think this calls for part of this standard to include any amendment applications that might be connected to an application, and potentially also a way to track which has been enacted? Definitely needs some digging, I think.

That's all we've got at the mo, sorry it's a bit wordy - shout if you would like to discuss anything in further detail!
